pynamodb/connection/table.py

```diff
@@ -139,6 +139,7 @@ def put_item(
         return_values: Optional[str] = None,
         return_consumed_capacity: Optional[str] = None,
         return_item_collection_metrics: Optional[str] = None,
+        return_values_on_condition_failure: Optional[str] = None,
```

Comment:
This could be typed as:

```suggestion
        return_values_on_condition_failure: Optional[Literal["ALL_OLD", "NONE"]] = None,
```